plugins/vpc-bridge/config/k8s/pipe.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+//go:build enablek8sconnector && windows
+
+package k8s
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"time"
+
+	winio "github.com/Microsoft/go-winio"
+	log "github.com/cihub/seelog"
+	"golang.org/x/sys/windows"
+)
+
+const (
+	//PipeConnectionTimeout Represents timeout for reading from connection object.
+	PipeConnectionTimeout = 1 * time.Minute
+
+	//NamedPipePathPrefix Represents prefix for named pipe path.
+	NamedPipePathPrefix = `\\.\pipe\aws-vpc-k8s-connector-%s`
```

Comment:
We should try to see if `\\.\pipe\ProtectedPrefix\Administrators\aws-vpc-k8s-connector-%s` works in this case. That would prevent non-admins from seeing the pipe which is a better security posture

Reply:
done and tested, it works.